src/components/atoms/SidebarItem.tsx

```diff
@@ -6,15 +6,18 @@ import Text from './Text'
 
 export type SidebarItemProps = {
   text: string
-  to: string
+  to: number
```

Comment:
```suggestion
  sectionId: number
```

요거 맥락상 `SectionId`임을 명확히 나타내주면 좋을 것 같아유! 다른 곳에서는 `to: string`으로 이동할 URL 자체를 받고 있는 곳도 있어서 헷갈리는 것 같습니다! 🙏